android-auto/build.gradle.kts

```diff
@@ -0,0 +1,33 @@
+plugins {
+    id("com.android.library")
+    kotlin("android")
+    kotlin("android.extensions")
+    kotlin("kapt")
+}
+
+android {
+    compileSdkVersion(30)
+
+    defaultConfig {
+        minSdkVersion(23)
+        missingDimensionStrategy("settings_visibility", "internal")
+    }
+
+    testOptions {
+        unitTests.isIncludeAndroidResources = true
+    }
+}
+
+dependencies {
+    api("com.mapbox.navigation:android:2.0.0-beta.21")
+    api("com.mapbox.search:mapbox-search-android:1.0.0-beta.17")
+    implementation("androidx.appcompat:appcompat:1.3.1")
+    implementation("androidx.car.app:app:1.0.0")
```

Comment:
Yeah I agree. And if we made it in the same style as 1tap-android we would also eliminate a step for copying things over.

But also, one of the "requirements" of this module at the moment, is to be completely decoupled from the app.

For example, we will be building instructions for adding this "starter-pack" to your repository. It will look something like this:
```
1. cd mapbox-navigation-android-examples
2. cp -fR android-auto/ ../{your-application}/android-auto
3. Update build.gradle.kts to include correct dependencies
```